services/jetbrains/jetbrains-rating.tester.js

```diff
@@ -33,52 +41,32 @@ t.create('rating stars (number as a plugin id)')
   .get('/stars/11941.json')
   .expectBadge({ label: 'rating', message: isStarRating })
 
-t.create('rating stars for unknown plugin')
+t.create('rating stars for unknown plugin (string)')
   .get('/stars/unknown-plugin.json')
   .expectBadge({ label: 'rating', message: 'not found' })
 
+t.create('rating stars for unknown plugin (numeric)')
+  .get('/stars/9999999999999.json')
+  .expectBadge({ label: 'rating', message: 'not found' })
+
+t.create('rating stars for unknown plugin (mixed)')
+  .get('/stars/9999999999999-abc.json')
+  .expectBadge({ label: 'rating', message: 'not found' })
+
 t.create('rating number')
   .get('/rating/11941.json')
-  .intercept(
-    nock =>
-      nock('https://plugins.jetbrains.com')
-        .get('/plugins/list?pluginId=11941')
-        .reply(
-          200,
-          `<?xml version="1.0" encoding="UTF-8"?>
-            <plugin-repository>
-              <category name="User Interface">
-                <idea-plugin downloads="1714" size="208537" date="1586449109000" updatedDate="1586449109000" url="">
-                  <rating>4.5</rating>
-                </idea-plugin>
-              </category>
-            </plugin-repository>`
-        ),
-    {
-      'Content-Type': 'text/xml;charset=UTF-8',
-    }
+  .intercept(nock =>
+    nock('https://plugins.jetbrains.com')
+      .get('/api/plugins/11941/rating')
+      .reply(200, { meanRating: 4.4848 })
   )
   .expectBadge({ label: 'rating', message: '4.5/5' })
 
 t.create('rating stars')
   .get('/stars/11941.json')
-  .intercept(
-    nock =>
-      nock('https://plugins.jetbrains.com')
-        .get('/plugins/list?pluginId=11941')
-        .reply(
-          200,
-          `<?xml version="1.0" encoding="UTF-8"?>
-            <plugin-repository>
-              <category name="User Interface">
-                <idea-plugin downloads="1714" size="208537" date="1586449109000" updatedDate="1586449109000" url="">
-                  <rating>4.5</rating>
-                </idea-plugin>
-              </category>
-            </plugin-repository>`
-        ),
-    {
-      'Content-Type': 'text/xml;charset=UTF-8',
-    }
```

Comment:
Same observation/question as the other tester around whether it'd make sense to maintain a mocked test 

(note I'm not saying that we must keep these, just wondering if it'd be beneficial to do so)